homeassistant/helpers/template.py

```diff
@@ -916,6 +916,29 @@ def device_entities(hass: HomeAssistant, _device_id: str) -> Iterable[str]:
     return [entry.entity_id for entry in entries]
 
 
+def integration_entities(hass: HomeAssistant, entry_name: str) -> Iterable[str]:
+    """
+    Get entity ids for entities tied to an integration/domain.
+
+    Provide entry_name as domain to get all entity id's for a integration/domain
+    or provide a config entry title for filtering between instances of the same integration.
+    """
+    conf_entry = next(
+        (
+            entry.entry_id
+            for entry in hass.config_entries.async_entries()
+            if entry_name == entry.title
+        ),
+        None,
+    )
+    return [
+        entity_id
+        for entity_id, info in hass.data.get("entity_info", {}).items()
```

Comment:
Don't directly access this. Instead use `entity_sources` imported from `helpers.entity`

Reply:
yeah, tried that but got me into a hell of circular imports, see 1 commit back...

Reply:
then import inside this function.